docs/pkglists/openSUSE_Leap_15.4.x86_64

```diff
@@ -1,52 +0,0 @@
-debug/x86_64/bareos-bconsole-debuginfo
```

Comment:
If this one is removed, shouldn't be an another file covering both SLE15 and openSUSE15 exists, or the decision is to use existing SLE_15_SP4 ?

Reply:
Yes, there is the file SUSE_15.x86_64 for this.